### PR TITLE
Use %n for new line in case of String.format

### PR DESCRIPTION
**Pull request by rsvoboda**
Use %n for new line in case of String.format

### CODE REVIEW

**Review by aguibert — APPROVED** [review]
LGTM

**Review by machi1990 — APPROVED** [review]
LGTM, thanks.

File: independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanInfo.java [hunk 1]
```diff
@@ -442,7 +442,7 @@ protected String getType() {
                     candidates, classLevelBindings);
             if (!finalMethods.isEmpty()) {
                 errors.add(new DeploymentException(String.format(
-                        "Intercepted methods of the bean %s may not be declared final:\n\t- %s", getBeanClass(),
+                        "Intercepted methods of the bean %s may not be declared final:%n\t- %s", getBeanClass(),
```
Comment (gsmet) [inline independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanInfo.java]: Not convinced by this one. It makes things less readable IMHO.

**Comment by rsvoboda** [comment]
@gsmet change for BeanInfo reverted

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**